docs/tasks/issuers/setup-venafi.rst

```diff
@@ -129,7 +129,8 @@ create a Kubernetes Secret resource containing your Venafi TPP API credentials:
    resource namespace'.
 
 These credentials will be used by cert-manager to interact with your Venafi TPP
-instance.
+instance.  Username attribute must be adhere to the <identity provider>:<username> format.  
```

Comment:
I've not actually noticed this requirement, and in our testing we use the username `tppadmin` without the `local` prefix.  Perhaps this is being defaulted somewhere in some cases? What other examples of identity providers could be used here? Can we maybe link to some Venafi docs on this too? 😄

Reply:
Ping ^ have you got any more info on the different identity providers that venafi supports?

Reply:
I'm a Solution Architect at Venafi and have access to the documentation, and this is pulled from the online WebSDK docs:  

> POST Authorize

> Authenticates a user via a username and password. The identity can originate from an identity provider, such as Active Directory, LDAP, or Local.

I may have hit a false positive on the local auth use case however there may be instances where AD auth may be necessary.  Example username would then be `AD+domain:username`